internal/metrics/metrics.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
```

Comment:
It maybe useful to comment about necessity of the metrics

Reply:
We'll be updating these metrics when we know more another time, for now the description of the metrics should do